url-updater/src/main/java/com/devonfw/tools/ide/url/folderhandling/UrlStatusFile.java

```diff
@@ -0,0 +1,77 @@
+package com.devonfw.tools.ide.url.folderhandling;
+
+import java.io.IOException;
+
+import com.devonfw.tools.ide.url.folderhandling.abstractUrlClasses.AbstractUrlFile;
+import com.devonfw.tools.ide.url.folderhandling.jsonfile.UrlJsonCompleteDataBlock;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+/**
+ * {@link UrlFile} for the "status.json" file.
+ */
+public class UrlStatusFile extends AbstractUrlFile {
+
+	/** Constant {@link UrlStatusFile#getName() filename}. */
+	static final String STATUS_JSON = "status.json";
+	private UrlJsonCompleteDataBlock jsonFileData;
+
+	private static final ObjectMapper MAPPER;
+	static {
+		MAPPER = new ObjectMapper();
+		MAPPER.registerModule(new JavaTimeModule());
+		MAPPER.enable(SerializationFeature.INDENT_OUTPUT);
+	}
+
+	/**
+	 * The constructor.
+	 *
+	 * @param parent the {@link #getParent() parent folder}.
+	 */
+	public UrlStatusFile(UrlVersion parent) {
+
+		super(parent, STATUS_JSON);
+	}
+
+	/**
+	 * @return The variable {@link UrlJsonCompleteDataBlock jsonFileData}.
+	 * @see {@link #doLoad()} loads the content of the json file (represented by
```

Comment:
Lift is correct here. What you really want is this:
```
@see #doLoad()
```
If you really wanted to include all the details then do something like this:
```
Also see {@link #doLoad()} that loads the content of the JSON file ...
```
However, in the later case consider that you are explaining what `doLoad` does and that should go to the JavaDoc of `doLoad` itself as otherwise you might end up in a lot of redundancy (e.g. when you are "linking" from multiple places to `doLoad` and that is why @see works exactly the way it is.